packages/client/src/v2-events/components/forms/utils.ts

```diff
@@ -14,7 +14,8 @@ import {
   Inferred,
   FieldValue,
   MetaFields,
-  isFieldConfigDefaultValue
+  isFieldConfigDefaultValue,
+  mapFieldTypeToEmptyValue
```

Comment:
this commit is to address this 
![image](https://github.com/user-attachments/assets/d555e243-82e9-46ab-a380-49eea6567697)